src/components/Bookings.js

```diff
@@ -0,0 +1,53 @@
+/* I had help with task: L3 T19 */
+
+import React from "react";
+import Search from "./Search";
+import SearchResults from "./SearchResults";
+import BookingNew from "./BookingNew";
+
+import useFetch from "./useFetch";
+
+const Bookings = () => {
+  const { currentData, setNewBooking, isPending, error, search } = useFetch(
+    "https://cyf-react.glitch.me"
+  );
```

Comment:
I really like this way of doing this, using a custom hook really demonstrates a good understanding of hooks.